SIGs/SIG-community/proposal.md

```diff
@@ -0,0 +1,24 @@
+# Proposal to create SIG Community
+
+This document is a proposal to create SIG-Community as a formal Special Interest Group (SIG) under the auspices of the TSC of the Bytecode Alliance, as specified in the TSC's charter.
+
+SIG-Community's goal is to create outreach workflows for the Bytecode Alliance ecosystem. The scope of the SIG focuses on driving ecosystem-wide outreach activities such as events, newsletters, case studies rather than focusing on project-specific ones. However, the SIG may give recommendations to individual projects on how to best tailor their efforts for consistency within the Bytecode Alliance.
+
+The group may also create open source tooling to help drive these efforts such as GitHub bots etc.
+
+As an immediate deliverable of the SIG, the group will be responsible for creating, populating, and maintaining an activity calendar for the Bytecode Alliance for the current year. Following this, the group will then direct efforts toward strategizing outreach for the ecosystem via newsletters, case studies, and other social media activities. 
+
+The group will communicate asynchronously (on Zulip) and synchronously through regular Zoom meetings to discuss current initiatives.
+
+## Bootstrap Members
+- Divya Mohan (Suse)
+- Karen Chu (Fermyon)
+
+*As the SIG's efforts grow, we are looking for people to jump in and help with new areas.*
+
+
+## Supporting Members
+
+The following individuals support the creation of SIG-Documentation:
```

Comment:
Eep also, I realized I had not amended this. Sorry Karen 😅

```suggestion
The following individuals support the creation of SIG-Community:
```

Reply:
Ooh! I totally missed that too. Thanks for catching it!